examples/python/cli/cli.py

```diff
@@ -178,7 +181,7 @@ def send_payment(params):
     sdk = Sdk(params.network)
     try:
         # Prepare the send request to get fees
-        prepare_req = breez_sdk_liquid.PrepareSendRequest(params.destination, params.amount)
+        prepare_req = breez_sdk_liquid.PrepareSendRequest(params.destination, PayAmount.RECEIVER(params.amount))
```

Comment:
The `params.amount` is optional here, so we should check its set before using `PayAmount.RECEIVER`